321. Clean space values in the image href

ebenezer-melkamu opened the pull request:
Fixes #321 

### Modifications 

1. Added a function to replace space chars with `%20`

ebenezer-melkamu committed:
Clean space values in the image href

IanMayo commented:
Nice work @AbegaM 

But, how about committing a modified image from `data` sub-folder?   Once the transform is complete, and the output is generated in `target/html`, we should see a corresponding change in the `dita` file that referred to that image URL (that now includes one or more %20 chars.

ebenezer-melkamu committed:
Modified image file for testing

ebenezer-melkamu commented:
@IanMayo I have modified the `daf_1.jpeg` image to `daf 1.jpeg` for testing, and i have modified the `Britain.html` file and its all working now, please let me know when you can test it

IanMayo commented:
> @IanMayo I have modified the `daf_1.jpeg` image to `daf 1.jpeg` for testing, and i have modified the `Britain.html` file and its all working now, please let me know when you can test it

That's perfect.  We can see the modified `DITA` page, and then the modified html output.
